### PR TITLE
Test py2/3 compatibility on Travis and minor fixes

### PR DESCRIPTION
**Pull request by ctk3b**
Accidentally included #204. 

The desmond parser had some minor issues with py3 compatibility. The main thing I need you to check @mrshirts is that I didn't mangle the `load_box_vector` function.

### CODE REVIEW

File: intermol/desmond/desmond_parser.py [hunk 1]
```diff
@@ -1080,22 +1080,20 @@ def load_box_vector(self, lines, start, end):
...
 
-        i = start
-        v = np.zeros([3,3])*units.angstroms
-        while (i<end):
-            if self.atom_box_vars[0] in lines[i]:
-                startboxlabel = i-start
-            if ':::' in lines[i]:
-                endlabel = i
+        v = np.zeros([3, 3]) * units.angstroms
+        import pdb; pdb.set_trace()
```
Comment (mrshirts) [inline intermol/desmond/desmond_parser.py]: Should get rid of this.

**Comment by ctk3b** [comment]
This should also let us test the conversion itself to avoid the most egregious failures.

File: intermol/desmond/desmond_parser.py [hunk 2]
```diff
@@ -1080,22 +1080,20 @@ def load_box_vector(self, lines, start, end):
...
+                endlabel = i + start
                 break
-            i+=1
-        startbox = startboxlabel+endlabel
-        nvec = 0
-        for i in range(startbox,startbox+9):
-            j = (nvec)/3
-            k = (nvec)%3
-            v[j,k] = float(re.sub(r'\s', '', lines[i])) * units.angstrom
-            nvec += 1
```
Comment (mrshirts) [inline intermol/desmond/desmond_parser.py]: Hard to tell if it's right without running, but if it's printing the same lines, it's probably good (and will be an easy fix later if loses some generalizability)

**Comment by mrshirts** [comment]
Seems to work in 2.7 fine.

**Pull request merged**